data-link/Cargo.toml

```diff
@@ -13,7 +13,7 @@ log = { version = "0.4.17", features = ["release_max_level_off"] }
 serde_json = "1.0.82"
 serde = { version = "1.0.138", features = ["derive"] }
 url = { version = "2.2.2", features = ["serde"] }
-reqwest = "0.11.11"
+reqwest = { version = "0.11.11", features = [ "json", "rustls-tls" ], default-features = false }
```

Comment:
@kirillt, this makes `OpenSSL` not required when compiling/building the project.

It's related to [Can't cross-compile project with OpenSSL](https://users.rust-lang.org/t/cant-cross-compile-project-with-openssl/70922/3)